_docs-sources/refarch/configuration/preflight-checks.md

```diff
@@ -0,0 +1,36 @@
+# Step 2. Iterate on Preflight checks 
+
+Once you have run the bootstrap script and pushed your `ref-arch-form` branch with your changes, your `infrastructure-live` repository GitHub actions will commence, running 
+preflight checks against every commit that you push up to this branch. 
+
+![Gruntwork Reference Architecture preflight checks](/img/preflight1.png)
+
+From the moment that your git commit is pushed, it usually takes around 4 minutes for your preflight check errors to appear on your pull request. Note that any errors will be 
+directly annotated on the exact line of your form that presents a blocking issue, so be sure to check the *Files changed* tab of your pull request to see them:
+
+![Gruntwork Ref Arch preflight checks on your pull request](/img/preflight-error-on-pr.png)
+
+## Fix up the errors 
+
+In most cases, the error messages included in the preflight check annotations will provide sufficient information to remediate the underlying issue. If at any point you are confused or 
+need assistance, please don't hesitate to reach out to us at `support@gruntwork.io` and we'll be happy to assist you. 
+
+## Commit your form changes and push to your `ref-arch-form` branch
+
+Once you have fixed the issues flagged by preflight checks, you can make a new commit with your latest form changes and push it up to the same branch. This will trigger a re-run of preflight
```

Comment:
```suggestion
Once you have fixed any issues flagged by preflight checks, you can make a new commit with your latest form changes and push it up to the same branch. This will trigger a re-run of preflight
```